editor/plugins/sprite_2d_editor_plugin.cpp

```diff
@@ -678,14 +709,20 @@ void Sprite2DEditorPlugin::make_visible(bool p_visible) {
 		sprite_editor->options->show();
 	} else {
 		sprite_editor->options->hide();
+		dragging_mode_hint->hide();
 		sprite_editor->edit(nullptr);
 	}
 }
 
 Sprite2DEditorPlugin::Sprite2DEditorPlugin() {
 	sprite_editor = memnew(Sprite2DEditor);
 	EditorNode::get_singleton()->get_gui_base()->add_child(sprite_editor);
-	make_visible(false);
+	sprite_editor->connect("theme_changed", callable_mp(this, &Sprite2DEditorPlugin::_editor_theme_changed));
+
+	dragging_mode_hint = memnew(Label);
+	_editor_theme_changed();
```

Comment:
Not needed, because the signal connected above is emitted when the node is added to scene tree.